host/3.0/buster/amd64/base/host.Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN EXTENSION_BUNDLE_VERSION=1.8.1 && \
     mkdir -p /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION && \
     unzip /$EXTENSION_BUNDLE_FILENAME -d /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION && \
     rm -f /$EXTENSION_BUNDLE_FILENAME && \
-    EXTENSION_BUNDLE_VERSION_V2=2.8.3 && \
+    EXTENSION_BUNDLE_VERSION_V2=2.6.1 && \
```

Comment:
I'm curious about this downgrade. Could you share the context?

Reply:
good catch @TsuyoshiUshio  - this is a recent update . PR is needs to be udpated.